pkg/kubectl/cmd/edit.go

```diff
@@ -193,29 +204,20 @@ func RunEdit(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []strin
 				os.Remove(results.file)
 			}
 
-			glog.V(4).Infof("User edited:\n%s", string(edited))
-			lines, err := hasLines(bytes.NewBuffer(edited))
-			if err != nil {
-				return preservedFile(err, file, out)
-			}
 			// Compare content without comments
 			if bytes.Equal(stripComments(original), stripComments(edited)) {
-				if len(results.edit) > 0 {
-					preservedFile(nil, file, out)
```

Comment:
This preservedFile call and the following are removed since they are redundant. We made no changes to the configuration or we did an empty save, there is no reason to preserve.